vo_models/vosi/tables/models.py

```diff
@@ -0,0 +1,18 @@
+"""Pydantic-xml models for the VOSI Tables specification"""
+from vo_models.vodataservice import Table, TableSet
+
+NSMAP = {
+    "vosi": "http://www.ivoa.net/xml/VOSITables/v1.0",
+    "vr": "http://www.ivoa.net/xml/VOResource/v1.0",
+    "vs": "http://www.ivoa.net/xml/VODataService/v1.1",
+    "xsd": "http://www.w3.org/2001/XMLSchema",
+    "xsi": "http://www.w3.org/2001/XMLSchema-instance",
+}
+
+
+class VOSITable(Table, tag="table", ns="vosi", nsmap=NSMAP):
```

Comment:
This is being rendered in the docs. Should it be? It has no docstring so there's no info there about the params. also `VOSITableSet`

https://vo-models.readthedocs.io/v0.3.x/pages/api/vosi_api.html#module-vo_models.vosi.tables.models

Reply:
They both should be, although I was somewhat hoping they'd inherit their displayed info from Table and TableSet respectively, since they are the same, just namespaced to VOSI. I'll have to dig into that.